playbooks/k8s-cluster/nvidia-network-operator.yaml

```diff
@@ -0,0 +1,12 @@
+---
+## Playbook for installing nvidia-network-operator

```

Comment:
Can you add some additional details as to why you would want this, what it does, and how it impacts the cluster?

Alternatively, should we add a blurb about this in the main k8s-cluster README file?

Reply:
I'm working on a README file and should be able to check in shortly.

Reply:
Okay, just left a bunch of comments there. A few spelling/grammar/styleguide mistakes, but also I don't think we need to duplicate the K8s deployment guide. I would still like to see a small update made to the k8s doc and an link to where the new network resource are defined in official nvidia docs.